src/components/TrackTile/TrackTile.test.tsx

```diff
@@ -94,7 +94,11 @@ describe('Track Tile', () => {
 
     const { findByLabelText } = render(
       <TrackTileProvider>
-        <TrackTile onOpenSettings={onOpenSettings} onOpenTracker={jest.fn()} />
+        <TrackTile
+          onOpenSettings={onOpenSettings}
+          onOpenTracker={jest.fn()}
+          title="Test Title"
+        />
       </TrackTileProvider>,
```

Comment:
This test alerted me to the fact that the settings icon/button on the TrackTile is now dependent on their being a `title` prop.  Both of these features are not yet tested in this PR and is still an open question as to how they should look when enabled.  I think @sternetj might have had an idea about how to do this.  In any case it will get it's own subsequent PR.